doc/source/dev/contributor/meson.rst

```diff
@@ -187,15 +175,15 @@ it does mean that one current workflow is no longer supported.
 The recommended workflow is to use ``python dev.py``. This works exactly the
 same way as ``python runtests.py`` worked before. What it does is rebuild if
 needed, and then install SciPy to a private directory (default is
-``installdir/`` in-tree) before running tests or other development tasks. This
+``build-install/`` in-tree) before running tests or other development tasks. This
 way modifications to pure Python code get picked up.
 
 If you use an IDE with, e.g., a "Run" button for scripts which were pointing to
 an in-place build, and you would really like to continue using that same
 workflow instead of ``python dev.py``, then you have a few options:
 
 - After modifying pure Python code in the SciPy repo, install it on the command
-  line with ``python dev.py --only-build``, or with ``meson install -C build``
+  line with ``python dev.py build``, or with ``meson install -C build``
   before running your script.
 - If your IDE supports it, customize what the "Run" button does before running
   the script, to do the install each time (this is expected to take 2-3 sec.)
```

Comment:
This only works if you export the PYTHONPATH somehow. Until we have in-place build, we should add a note about this.

Reply:
added a note on this